apps/core/src/hooks/useUnlockTimelockedObjectsTransaction.ts

```diff
@@ -4,17 +4,19 @@
 import { useIotaClient } from '@iota/dapp-kit';
 import { createUnlockTimelockedObjectsTransaction } from '../utils';
 import { useQuery } from '@tanstack/react-query';
+import { useMaxTransactionSizeBytes } from './useMaxTransactionSizeBytes';
 
 export function useUnlockTimelockedObjectsTransaction(address: string, objectIds: string[]) {
     const client = useIotaClient();
+    const { data: maxTxSizeBytes = Infinity } = useMaxTransactionSizeBytes();
```

Comment:
just a nit: we can change this var name `maxTxSizeBytes`  to `maxSizeBytes`
 ad than use it   >        `  await transaction.build({ client, maxSizeBytes });`